Makefile

```diff
@@ -42,6 +46,14 @@ build/properties.owl: src/properties.tsv | build/robot.jar
 build/cineca.owl: build/properties.owl build/cineca.tsv | build/robot.jar
 	$(ROBOT) template --input $< --merge-before --template $(word 2,$^) --output $@
 
+build/ncit-terms.txt: build/cineca.owl src/cineca/get-ncit-ids.rq src/cineca/ncit-annotation-properites.txt | build/robot.jar
+	$(ROBOT) query --input $< --query $(word 2,$^) $@
+	tail -n +2 $@ > $@.tmp
+	cat $@.tmp $(word 3,$^) > $@ && rm $@.tmp
+
+build/ncit-module.owl: build/ncit-terms.txt | build/robot-rdfxml.jar
+	$(ROBOT_RDFXML) extract --input ncit.owl --term-file $< --method rdfxml --intermediates minimal --output $@
```

Comment:
Please tell Make to download `build/ncit.owl` and add it to the inputs for this task.